manifests/charts/gateways/istio-egress/templates/deployment.yaml

```diff
@@ -228,6 +228,11 @@ spec:
           - name: {{ $key }}
             value: "{{ $value }}"
           {{- end }}
+          {{- $network_set := index $gateway.env "ISTIO_META_NETWORK" }}
```

Comment:
This is a bit tricky. Without this, the PR broke some security tests. I have no clue how it possibly passed before. The test deploys a egress gateway, which has no network configured, then it deploys some pods without sidecars and tries to send traffic to it. So egress gets no endpoints for this pod.

This aligns the egress chart with ingress

Reply:
Ah figured it out! We are missing the namespace label on istio-system, since the test sets up istio-system *before* calling the istio setup code, leading to `failed creating namespace istio-system` and not setting network properly.

So it was a bug in the test. This fixes the test bug, revealing the real bug, which is fixed here